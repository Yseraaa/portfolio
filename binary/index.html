<!DOCTYPE html><html lang="en"><head>   <meta charset="UTF-8">   <title>binary</title>   <link rel="stylesheet" href="css/style.min.css">   <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700;900&display=swap" rel="stylesheet"></head><body>   <div class="header">      <div class="header__logo">         <a href="#"><picture><source srcset="img/logo.webp" type="image/webp"><img class='header__logo_img' src="img/logo.png" alt="logo"></picture></a>      </div>      <div class="header__text">         <p class="h_text1">Clear, smart, attractive design </p>         <p class="h_text2">for your app, logo, website</p>      </div>      <div class="header__line"><picture><source srcset="img/Line.webp" type="image/webp"><img src="img/Line.png" alt="line"></picture></div>      <center>         <div class="header__button">            <a href="">               <p class="h_button">READ MORE</p>            </a>         </div>      </center>      <div class="header__arrowdown">         <a href="#"><picture><source srcset="img/Arrow-down.webp" type="image/webp"><img src="img/Arrow-down.png" alt="arrow"></picture></a>      </div>   </div>   <div class="nav">      <div class="container"> <a class="nav_link nav_link_active" href="#">HOME</a> <a class="nav_link" href="#">ABOUT            US</a> <a class="nav_link" href="#">PORTFOLIO</a> <a class="nav_link" href="#">CONTACT</a> </div>   </div>   <div class="slider1">      <div class="slider1__about">         <p>ABOUT</p>      </div>      <div class="slider1__desc">         <p>We specialize in contemporary, functional, and stunning websites with design that really brings that "WOW"            factor.</p>         <p>In our design process, we make sure to stay one step ahead of the latest trends,</p>         <p>never forgetting to abide by key web standards</p>      </div>      <div class="slider1__apps">         <div class="slider1-about">            <div class="slider1-about__icon"> <picture><source srcset="img/icon/icon1.webp" type="image/webp"><img src="img/icon/icon1.png" alt="" /></picture> </div>            <div class="slider1-about__title">WEB</div>            <div class="slider1-about__text">               <p>We tend to deliver unique and</p>               <p>functional design, through </p>               <p>innovation and creativity.</p>            </div>         </div>         <div class="slider1-about">            <div class="slider1-about__icon"> <picture><source srcset="img/icon/icon2.webp" type="image/webp"><img src="img/icon/icon2.png" alt="" /></picture> </div>            <div class="slider1-about__title">iOS AND ANDROID</div>            <div class="slider1-about__text">               <p>We seek to bring new businesses and </p>               <p>customers closer together through valuable </p>               <p>insights and proper targeting.</p>            </div>         </div>         <div class="slider1-about">            <div class="slider1-about__icon"> <picture><source srcset="img/icon/icon3.webp" type="image/webp"><img src="img/icon/icon3.png" alt="" /></picture> </div>            <div class="slider1-about__title">MARKETING</div>            <div class="slider1-about__text">               <p>Weâ€™ll help you establish the right goals </p>               <p>and build the right paths. Success depends </p>               <p>on setting the right intention.</p>            </div>         </div>      </div>   </div>   <div class="slider2">      <center>         <div class="slider2__portf">PORTFOLIO</div>      </center>      <div class="slider2__nav"><a class='slider2_link nav_link_active' href="#">ALL</a> <a class='slider2_link'            href="#">WEB</a> <a class='slider2_link' href="#">APPS</a> <a class='slider2_link' href="#">OTHER</a></div>      <div class="slider2__items">         <a href="#">            <div class="slider2__block"></div>         </a>         <a href="#">            <div class="slider2__block"></div>         </a>         <a href="#">            <div class="slider2__block"></div>         </a>         <a href="#">            <div class="slider2__block"></div>         </a>         <a href="#">            <div class="slider2__block"></div>         </a>         <a href="#">            <div class="slider2__block"></div>         </a>      </div>      <center>         <a href="#">            <div class="slider2__button">READ MORE</div>         </a>      </center>   </div>   <div class="slider3">      <div class="slider3__name">         <p>Our partners</p>      </div>      <div class="slider3__items">         <a href="#"><picture><source srcset="img/icon/icon4.webp" type="image/webp"><img class='s3item' src="img/icon/icon4.png" alt="4"></picture></a>         <a href="#"><picture><source srcset="img/icon/icon5.webp" type="image/webp"><img class='s3item' src="img/icon/icon5.png" alt="5"></picture></a>         <a href="#"><picture><source srcset="img/icon/icon6.webp" type="image/webp"><img class='s3item' src="img/icon/icon6.png" alt="6"></picture></a>         <a href="#"><picture><source srcset="img/icon/icon7.webp" type="image/webp"><img class='' src="img/icon/icon7.png" alt="7"></picture></a>      </div>   </div>   <div class="contact">      <center>         <div class="contact__cont">CONTACT</div>      </center>      <div class="contact__all">         <div class="contact__left">            <form action="#">               <div class="cont__form">                  <div class="cont__name">YOUR NAME                     <input class='marginform' type="text"> </div>                  <div class="cont__email">YOUR E-MAIL                     <input class='marginform' type="text"> </div>               </div>               <div class="cont__msg"> MESSAGE                  <input class='formsg' type="message"> </div>            </form>            <div class="contact__send"><a href="#">SEND</a></div>         </div>         <div class="contact__text">            <p>Have an idea for a project? Just looking to</p>            <p>explore and see the possibilities?</p>            <p>I'm here to help.</p> <span>Postma Elena</span>            <div class="contact__social">               <div class="contact__item">                  <a href="#">                     <div class="contact__fb"><picture><source srcset="img/icon/Facebook.webp" type="image/webp"><img src="img/icon/Facebook.png" alt=""></picture></div>                  </a>               </div>               <div class="contact__item contact__item_margin">                  <a href="#">                     <div class="contact__in"><picture><source srcset="img/icon/in.webp" type="image/webp"><img src="img/icon/in.png" alt=""></picture></div>                  </a>               </div>            </div>            <div class="contact__email">               <p class='email-bold'>e-mail: <a href="#">postma.elena@gmail.com</a></p>            </div>         </div>      </div>   </div>   <div class="footer">      <div class="footer__logo"><picture><source srcset="img/footer_logo.webp" type="image/webp"><img class='footer__logo_img' src="img/footer_logo.png" alt=""></picture></div>      <div class="footer__nav"> <a class='footer__nav_link' href="#">Home</a> <a class='footer__nav_link'            href="#">Facebook</a> <a class='footer__nav_link' href="#">Linkedin</a> <a class='footer__nav_link'            href="#">Contact</a> </div>      <div class="footer__copy">&copy 2015. All Rights Reserved </div>   </div>   <script src="js/script.min.js"></script></body></html>