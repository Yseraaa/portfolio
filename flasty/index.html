<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <title>Flasty</title>
    <link rel="stylesheet" href="css/style.css">
    <link href="https://fonts.googleapis.com/css2?family=Cherry+Swash&display=swap" rel="stylesheet"> </head>

<body>
    <div class="container">
        <div class="header">
            <ul class="menu">
                <h1 class="logo"><a href="#">Black Sea</a></h1>
                <a href="#" class="menu-link">
                    <li class="menu-list apper">Home</li>
                </a>
                <a href="#" class="menu-link">
                    <li class="menu-list apper">About me</li>
                </a>
                <a href="#" class="menu-link">
                    <li class="menu-list active apper">Portfolio</li>
                </a>
                <a href="#" class="menu-link">
                    <li class="menu-list">Contact me</li>
                </a>
            </ul>
        </div>
        <div class="slider1">
            <h1 class="zagolovok">We need the tonic of <br>wildness</h1>
            <a class='buttona' href="#"><div class="button"><p class="buttontext">Read more</p></div></a>
            <div class="pagination">
                <a href="#"><div class="page"></div></a>
                <a href="#"><div class="page"></div></a>
                <a href="#"><div class="page pageactive"></div></a>
                <a href="#"><div class="page"></div></a>
                <a href="#"><div class="page"></div></a>
            </div>
        </div>
        <div class="slider2">
            <h2 class="headers2">At the same time that we are earnest to explore 
<br><b>&</b> learn all things</h2>
            <p class="desc2">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever <br> since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. 
</p>
            <div class="shapes">
                <a class='shapeh' href=""><img class='' src="img/Shape_2.svg" alt="shape1"></a>
                <a class='shapeh' href=""><img class='shape' src="img/Shape_3.svg" alt="shape2"></a>
                <a class='shapeh' href=""><img class='shape' src="img/Shape_4.svg" alt="shape3"></a>
            </div>
        </div>
        <div class="slider3">

            <p class="desk3">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to 
a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting
</p>
        </div>
        <div class="slider4">
            <div class="obr">
            <form action="svyz">
                <input class='name' type="Name" placeholder="Name">
                <input class='email' type="Email" placeholder="Email address">
                <textarea class='msg' name="msg" placeholder="Message"></textarea>
                <a class='sumbit' href="#"><img src="img/Shape_5.svg" alt="shape5"></a>
            </form>

        </div>

        </div> 
                <div class="footer">
                    <p class="footertext">Copyright &copy 2012 Css Author.com</p>
                </div>
        </div>

</body>

</html>