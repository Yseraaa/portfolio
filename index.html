<!DOCTYPE html><html lang="en"><head>   <meta charset="UTF-8">   <meta name="viewport" content="width=device-width, initial-scale=1.0">   <title>Allison Werner</title>   <link rel="stylesheet" href="css/style.min.css">   <link rel="stylesheet" href="css/animate.min.css">   <meta name="yandex-verification" content="f94a2a2877b18cef" />   <meta name="description" content="Allison Werner (Ysera) Website (just for test seo tags)">   <meta name="keywords" content="html css js freelance website">   <link rel="shortcut icon" href="/img/icon/favicon.ico" type="image/x-icon"></head><body><div class="preloader">   <svg class="preloader__image" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512">   <path fill="currentColor"      d="M304 48c0 26.51-21.49 48-48 48s-48-21.49-48-48 21.49-48 48-48 48 21.49 48 48zm-48 368c-26.51 0-48 21.49-48 48s21.49 48 48 48 48-21.49 48-48-21.49-48-48-48zm208-208c-26.51 0-48 21.49-48 48s21.49 48 48 48 48-21.49 48-48-21.49-48-48-48zM96 256c0-26.51-21.49-48-48-48S0 229.49 0 256s21.49 48 48 48 48-21.49 48-48zm12.922 99.078c-26.51 0-48 21.49-48 48s21.49 48 48 48 48-21.49 48-48c0-26.509-21.491-48-48-48zm294.156 0c-26.51 0-48 21.49-48 48s21.49 48 48 48 48-21.49 48-48c0-26.509-21.49-48-48-48zM108.922 60.922c-26.51 0-48 21.49-48 48s21.49 48 48 48 48-21.49 48-48-21.491-48-48-48z">   </path>   </svg></div>   <div class="container" id='top'>      <header class="header ">         <div class="header__logo wow animate__lightSpeedInLeft"><a href="#">Allison Werner</a></div>         <div class="header__body">            <div class="header__menu">               <span></span>            </div>         </div>      </header>      <div class="menu__active">         <ul class="main-menu">            <span></span>            <li class="menu-link link__active "><a href="#top"># Home</a></li>            <li class="menu-link "><a href="#social"># Social</a></li>            <li class="menu-link "><a href="#proj"># Projects</a></li>            <li class="menu-link "><a href="#cont"># Contact</a></li>         </ul>      </div>      <div class="slider1 ">         <div class="slider1__body">            <div class="slider1__left wow animate__backInLeft">               <p>HTML, CSS,</p>               <p>BOOTSTRAP, GIT, JS</p>               <p>and more...</p>            </div>            <div class="slider1__right wow animate__backInRight">               <div class="slider1__right__top"></div>               <div class="slider1__right__form">                  <form class='slider1__form' action="https://formspree.io/xgenodql" method="POST">                     <label>                        <input class='form_name' type="text" name="name" placeholder="Name"></label>                     <label>                        <input class='form_email' type="email" name="_replyto" placeholder="Email"></label>                     <label>                        <textarea class='form_msg' name="message" placeholder="Message"></textarea>                     </label>                     <label><input class='form_btn' type="submit" value="Send!"></label>                  </form>               </div>            </div>         </div>      </div>      <div class="slider2">         <div class="slider2__media" id='social'>            <div class="slider2__social__left wow animated animate__bounceInLeft">               <div class="social__name">Social Media</div>               <div class="social__text">Here is a text to attract attention, it does not carry any semantic load.</div>            </div>            <div class="slider2__social__right wow animated animate__slideInUp">               <a class='social__icons' href="https://instagram.com/yseraa/" target="_blank"><picture><source srcset="img/icon/ig.webp" type="image/webp"><imgsrc="img/icon/ig.png" alt=""></picture></a>               <a class='social__icons' href="https://t.me/xxdone" target="_blank"><picture><source srcset="img/icon/tg.webp"alt="" type="image/webp"><img src="img/icon/tg.png"alt=""></picture></a>               <a class='social__icons' href="https://github.com/Yseraaa" target="_blank"><picture><source srcset="img/icon/gh.webp"alt="" type="image/webp"><img src="img/icon/gh.png"alt=""></picture></a>               <a class='social__icons' href="https://vk.com" target="_blank"><picture><source srcset="img/icon/vk.webp" type="image/webp"><img src="img/icon/vk.png" alt=""></picture></a>               <a class='social__icons' href="https://instagram.com/yseraa_/" target="_blank"<picture><source srcset="img/icon/ig.webp" type="image/webp"><img src="img/icon/ig.png" alt=""></picture></a>               <picture><source srcset="img/icon/tw.webp" type="image/webp"><img src="img/icon/tw.png" alt=""></picture>            </div>         </div>         <div class="slider2__name">Some random images</div>         <div class="slider2__slick wow animated animate__slideInUp">            <div class="slider2__slick__item "><picture><source srcset="img/slider/1.webp" type="image/webp"><img src="img/slider/1.jpg" alt=""></picture></div>            <div class="slider2__slick__item "><picture><source srcset="img/slider/2.webp" type="image/webp"><img src="img/slider/2.jpg" alt=""></picture></div>            <div class="slider2__slick__item "><picture><source srcset="img/slider/3.webp" type="image/webp"><img src="img/slider/3.jpg" alt=""></picture></div>            <div class="slider2__slick__item "><picture><source srcset="img/slider/4.webp" type="image/webp"><img src="img/slider/4.jpg" alt=""></picture></div>            <div class="slider2__slick__item "><picture><source srcset="img/slider/5.webp" type="image/webp"><img src="img/slider/5.jpg" alt=""></picture></div>            <div class="slider2__slick__item "><picture><source srcset="img/slider/6.webp" type="image/webp"><img src="img/slider/6.jpg" alt=""></picture></div>            <div class="slider2__slick__item "><picture><source srcset="img/slider/7.webp" type="image/webp"><img src="img/slider/7.jpg" alt=""></picture></div>            <div class="slider2__slick__item "><picture><source srcset="img/slider/8.webp" type="image/webp"><img src="img/slider/8.jpg" alt=""></picture></div>            <div class="slider2__slick__item "><picture><source srcset="img/slider/9.webp" type="image/webp"><img src="img/slider/9.jpg" alt=""></picture></div>            <div class="slider2__slick__item "><picture><source srcset="img/slider/10.webp" type="image/webp"><img src="img/slider/10.jpg" alt=""></picture></div>            <div class="slider2__slick__item "><picture><source srcset="img/slider/11.webp" type="image/webp"><img src="img/slider/11.jpg" alt=""></picture></div>            <div class="slider2__slick__item "><picture><source srcset="img/slider/12.webp" type="image/webp"><img src="img/slider/12.jpg" alt=""></picture></div>            <div class="slider2__slick__item "><picture><source srcset="img/slider/13.webp" type="image/webp"><img src="img/slider/13.jpg" alt=""></picture></div>            <div class="slider2__slick__item "><picture><source srcset="img/slider/14.webp" type="image/webp"><img src="img/slider/14.jpg" alt=""></picture></div>         </div>      </div>      <div class="slider3" id='proj'>         <div class="slider3__name">Projects</div>         <div class="slider3__items">            <div class="slider3__item wow animate__fadeInUpBig">               <div class="slider3__item_name">Retrowear</div>               <div class="slider3__logo"><picture><source srcset="img/site_card/retrowear.webp" type="image/webp"><img src="img/site_card/retrowear.png" alt=""></picture></div>               <div class="slider3__more">More</div>            </div>            <div class="slider3__item wow animate__fadeInUpBig">               <div class="slider3__item_name">Binary</div>               <div class="slider3__logo"><picture><source srcset="img/site_card/binary.webp" type="image/webp"><img src="img/site_card/binary.png" alt=""></picture></div>               <div class="binary__btn">More</div>            </div>            <div class="slider3__item wow animate__fadeInUpBig">               <div class="slider3__item_name">Flasty</div>               <div class="slider3__logo"><picture><source srcset="img/site_card/flasty.webp" type="image/webp"><img src="img/site_card/flasty.png" alt=""></picture></div>               <div class="flasty__btn">More</div>            </div>            <div class="slider3__item wow animate__fadeInUpBig">               <div class="slider3__item_name">Golden-One</div>               <div class="slider3__logo"><picture><source srcset="img/site_card/golden-one.webp" type="image/webp"><img src="img/site_card/golden-one.png" alt=""></picture></div>               <div class="golden__btn">More</div>            </div>            <div class="slider3__item wow animate__fadeInUpBig">               <div class="slider3__item_name">H-Game</div>               <div class="slider3__logo"><picture><source srcset="img/site_card/h-game.webp" type="image/webp"><img src="img/site_card/h-game.png" alt=""></picture></div>               <div class="hgame__btn">More</div>            </div>            <div class="slider3__item wow animate__fadeInUpBig">               <div class="slider3__item_name">Interior</div>               <div class="slider3__logo"><picture><source srcset="img/site_card/interior.webp" type="image/webp"><img src="img/site_card/interior.png" alt=""></picture></div>               <div class="interior__btn">More</div>            </div>            <div class="slider3__item wow animate__fadeInUpBig">               <div class="slider3__item_name">Music Festival</div>               <div class="slider3__logo"><picture><source srcset="img/site_card/m-fest.webp" type="image/webp"><img src="img/site_card/m-fest.png" alt=""></picture></div>               <div class="mfest__btn">More</div>            </div>            <div class="slider3__item wow animate__fadeInUpBig">               <div class="slider3__item_name">Opoiki</div>               <div class="slider3__logo"><picture><source srcset="img/site_card/opoiki.webp" type="image/webp"><img src="img/site_card/opoiki.png" alt=""></picture></div>               <div class="opoiki__btn">More</div>            </div>            <div class="slider3__item wow animate__fadeInUpBig">               <div class="slider3__item_name">Vintage</div>               <div class="slider3__logo"><picture><source srcset="img/site_card/vintage.webp" type="image/webp"><img src="img/site_card/vintage.png" alt=""></picture></div>               <div class="vintage__btn">More</div>            </div>         </div>         <div class="slider3__popup_retro popup">            <div class="slider3__popup_name">Retrowear.ru</div>            <div class="slider3__popup_logo"><picture><source srcset="img/site_card/retrowear__full.webp" type="image/webp"><img src="img/site_card/retrowear__full.png" alt=""></picture></div>            <a href="https://retrowear.ru" target="_blank">               <div class="slider3__popup_btn">GO</div>            </a>            <p class="close">Close</p>         </div>         <div class="slider3__popup_binary popup">            <div class="slider3__popup_name">Binary</div>            <div class="slider3__popup_logo"><picture><source srcset="img/site_card/binary_full.webp" type="image/webp"><img src="img/site_card/binary_full.png" alt=""></picture></div>            <a href="/binary/index.html" target="_blank">               <div class="slider3__popup_btn">GO</div>            </a>            <p class="close">Close</p>         </div>         <div class="slider3__popup_flasty popup">            <div class="slider3__popup_name ">Flasty</div>            <div class="slider3__popup_logo"><picture><source srcset="img/site_card/flasty__full.webp" type="image/webp"><img src="img/site_card/flasty__full.png" alt=""></picture></div>            <a href="/flasty/index.html" target="_blank">               <div class="slider3__popup_btn">GO</div>            </a>            <p class="close">Close</p>         </div>         <div class="slider3__popup_golden popup">            <div class="slider3__popup_name ">Golden-One</div>            <div class="slider3__popup_logo"><picture><source srcset="img/site_card/golden-one_full.webp" type="image/webp"><img src="img/site_card/golden-one_full.png" alt=""></picture></div>            <a href="/golden/index.html" target="_blank">               <div class="slider3__popup_btn">GO</div>            </a>            <p class="close">Close</p>         </div>         <div class="slider3__popup_hgame popup">            <div class="slider3__popup_name ">H-Game</div>            <div class="slider3__popup_logo"><picture><source srcset="img/site_card/h-game__full.webp" type="image/webp"><img src="img/site_card/h-game__full.png" alt=""></picture></div>            <a href="/hgame/index.html" target="_blank">               <div class="slider3__popup_btn">GO</div>            </a>            <p class="close">Close</p>         </div>         <div class="slider3__popup_interior popup">            <div class="slider3__popup_name ">Interior</div>            <div class="slider3__popup_logo"><picture><source srcset="img/site_card/interior__full.webp" type="image/webp"><img src="img/site_card/interior__full.png" alt=""></picture></div>            <a href="/interior/index.html" target="_blank">               <div class="slider3__popup_btn">GO</div>            </a>            <p class="close">Close</p>         </div>         <div class="slider3__popup_mfest popup">            <div class="slider3__popup_name ">Music Festival</div>            <div class="slider3__popup_logo"><picture><source srcset="img/site_card/m-fest__full.webp" type="image/webp"><img src="img/site_card/m-fest__full.png" alt=""></picture></div>            <a href="/mfest/index.html" target="_blank">               <div class="slider3__popup_btn">GO</div>            </a>            <p class="close">Close</p>         </div>         <div class="slider3__popup_opoiki popup">            <div class="slider3__popup_name">Opoiki</div>            <div class="slider3__popup_logo"><picture><source srcset="img/site_card/opoiki__full.webp" type="image/webp"><img src="img/site_card/opoiki__full.png" alt=""></picture></div>            <a href="/opoiki/index.html" target="_blank">               <div class="slider3__popup_btn">GO</div>            </a>            <p class="close">Close</p>         </div>         <div class="slider3__popup_vintage popup">            <div class="slider3__popup_name">Vintage</div>            <div class="slider3__popup_logo"><picture><source srcset="img/site_card/vintage__full.webp" type="image/webp"><img src="img/site_card/vintage__full.png" alt=""></picture></div>            <a href="/vintage/index.html" target="_blank">               <div class="slider3__popup_btn">GO</div>            </a>            <p class="close">Close</p>         </div>      </div>      <div class="slider4 wow animate__slideInUp" id='cont'>         <div class="slider4__name">Contact</div>         <div class="slider4__text">U can contact with me as email</div>         <div class="slider4__form_form">            <form class='slider4__form' action="https://formspree.io/xgenodql" method="POST">               <div class="slider4__form_all">                  <div class="slider4__form__left">                     <input class='form_name_cnt' type="text" name="name" placeholder="Name">                     <input class='form_email_cnt' type="email" name="_replyto" placeholder="Email">                     <input class='form_subject_cnt' type="subject" name="_subject" placeholder="Subject">                  </div>                  <div class="slider4__form_right">                     <textarea class='form_msg_cnt' name="message" placeholder="Message"></textarea>                  </div>               </div>               <input class='form_btn_cnt' type="submit" value="Send!">            </form>         </div>      </div>      <div class="footer">         <!-- Yandex.Metrika informer --><a href="https://metrika.yandex.ru/stat/?id=64316299&amp;from=informer"target="_blank" rel="nofollow"><picture><source srcset="https://informer.yandex.ru/informer/64316299/3_0_535353FF_333333FF_1_pageviews"style=" type="image/webp"><img src="https://informer.yandex.ru/informer/64316299/3_0_535353FF_333333FF_1_pageviews"style="width:88px; height:31px; border:0;" alt="Яндекс.Метрика" title="Яндекс.Метрика: данные за сегодня (просмотры, визиты и уникальные посетители)" class="ym-advanced-informer" data-cid="64316299" data-lang="ru" /></picture></a><!-- /Yandex.Metrika informer --><div class="footer__text">Alwerner.tk &copy; 2020</div>         <a href="#top"><div class="footer__up"><picture><source srcset="img/icon/arrow_up.webp" type="image/webp"><img src="img/icon/arrow_up.png" alt=""></picture></div> </a>      </div>   </div>   <div class="backpopup"></div>   <script src="js/jquery-3.5.1.min.js"></script>   <script src="js/wow.min.js"></script>   <script src="js/slick.min.js"></script>   <script src="js/smooth-scroll.polyfills.min.js"></script>   <script src="js/script.min.js"></script>   <!-- Yandex.Metrika counter -->   <script type="text/javascript">      (function (m, e, t, r, i, k, a) {      m[i] = m[i] || function () { (m[i].a = m[i].a || []).push(arguments) };         m[i].l = 1 * new Date(); k = e.createElement(t), a = e.getElementsByTagName(t)[0], k.async = 1, k.src = r, a.parentNode.insertBefore(k, a)      })         (window, document, "script", "https://mc.yandex.ru/metrika/tag.js", "ym");      ym(64316299, "init", {         clickmap: true,         trackLinks: true,         accurateTrackBounce: true,         webvisor: true      });   </script>   <!-- /Yandex.Metrika counter --></body></html>