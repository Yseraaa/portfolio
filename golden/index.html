<!DOCTYPE html>
<html lang="en">

<head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link rel="stylesheet" href="css/style.css">
   <title>Golden-One</title>
</head>

<body>
   <div class="container">
      <header>
         <div class="header">
            <nav>
               <div class="header__nav">
                  <div class="header__logo"><a href="#"><img src="img/logo.png" alt=""></a></div>
                  <div class="header__nav_menu">
                     <ul>
                        <li><a href="#">Home</a></li>
                        <li><a href="#">Services</a></li>
                        <li><a href="#">Portfolio</a></li>
                        <li><a href="#">About</a></li>
                        <li><a href="#">Contact</a></li>
                     </ul>
                  </div>

               </div>
            </nav>
            <div class="header__main">
               <div class="header__main_name">Welcome To Our Studio!</div>
               <div class="header__main_text">Itâ€™s nice to Meet you</div>
               <div class="header__main_btn">Tell Me More</div>
            </div>
         </div>
      </header>
      <div class="slider1">

         <div class="slider1__name">Services</div>
         <div class="slider1__text">Proin iaculis purus consequat sem cure. </div>
         <div class="slider1__main">
            <div class="slider1__main_block">
               <div class="slider1__main_block_logo"><img src="img/icons/icon1.png.png" alt=""></div>
               <div class="slider1__main_block_name">E-Commerce</div>
               <div class="slider1__main_block_text">
                  <p>Proin iaculis purus consequat sem cure</p>
                  <p>digni ssim. Donec porttitora entum suscipit</p>
                  <p>aenean rhoncus posuere odio in tincidunt.</p>
               </div>
            </div>
            <div class="slider1__main_block">
               <div class="slider1__main_block_logo"><img src="img/icons/icon1.png.png" alt=""></div>
               <div class="slider1__main_block_name">Responsive Web</div>
               <div class="slider1__main_block_text">
                  <p>Proin iaculis purus consequat sem cure</p>
                  <p>digni ssim. Donec porttitora entum suscipit</p>
                  <p>aenean rhoncus posuere odio in tincidunt.</p>
               </div>
            </div>
            <div class="slider1__main_block">
               <div class="slider1__main_block_logo"><img src="img/icons/icon1.png.png" alt=""></div>
               <div class="slider1__main_block_name">Web Security</div>
               <div class="slider1__main_block_text">
                  <p>Proin iaculis purus consequat sem cure</p>
                  <p>digni ssim. Donec porttitora entum suscipit</p>
                  <p>aenean rhoncus posuere odio in tincidunt.</p>
               </div>
            </div>
         </div>
      </div>
      <div class="slider2">
         <div class="slider2__name">Our portfolio</div>
         <div class="slider2__text">Proin iaculis purus consequat sem cure. </div>
         <div class="slider2__main">
            <div class="slider2__main_block"><img src="img/img1.jpg.jpg" alt=""></div>
            <div class="slider2__main_block"><img src="img/img2.jpg.png" alt=""></div>
            <div class="slider2__main_block"><img src="img/img3.jpg.png" alt=""></div>
         </div>

      </div>
      <div class="slider3">
         <div class="slider3__name">About Us</div>
         <div class="slider3__text">Proin iaculis purus consequat sem cure. </div>
         <div class="slider3__main">
            <div class="slider3__main_block">
               <div class="slider3__block_left">
                  <div class="slider3__main_block_name">
                     <p>July 2010</p>
                     <p>Our Humble Beginnings</p>
                  </div>
                  <div class="slider3__main_block_text">
                     <p>Proin iaculis purus consequat sem cure </p>
                     <p>digni ssim. Donec porttitora entum suscipit </p>
                     <p>aenean rhoncus posuere odio in tincidunt. Proin</p>
                     <p>iaculis purus consequat sem cure digni </p>
                     <p>ssim. Donec porttitora entum suscipit.</p>
                  </div>
               </div>
               <div class="slider3__main_right">
                  <div class="slider3__main_block_logo"><img src="img/slider3/about1.jpg.jpg" alt=""></div>
               </div>
            </div>
            <div class="slider3__main_block blockn2">
               <div class="slider3__main_right">
                  <div class="slider3__main_block_logo"><img src="img/slider3/about2.jpg.jpg" alt=""></div>
               </div>
               <div class="slider3__block_left">
                  <div class="slider3__main_block_name block2">
                     <p>January 2011</p>
                     <p>
                        Facing Startup Battles
                     </p>
                  </div>
                  <div class="slider3__main_block_text block2">
                     <p>Proin iaculis purus consequat sem cure </p>
                     <p>digni ssim. Donec porttitora entum suscipit </p>
                     <p>aenean rhoncus posuere odio in tincidunt. Proin</p>
                     <p>iaculis purus consequat sem cure digni </p>
                     <p>ssim. Donec porttitora entum suscipit.</p>
                  </div>
               </div>

            </div>
            <div class="slider3__main_block">
               <div class="slider3__block_left">
                  <div class="slider3__main_block_name">
                     <p>December 2012

                     </p>
                     <p>Enter The Dark Days</p>
                  </div>
                  <div class="slider3__main_block_text">
                     <p>Proin iaculis purus consequat sem cure </p>
                     <p>digni ssim. Donec porttitora entum suscipit </p>
                     <p>aenean rhoncus posuere odio in tincidunt. Proin</p>
                     <p>iaculis purus consequat sem cure digni </p>
                     <p>ssim. Donec porttitora entum suscipit.</p>
                  </div>
               </div>
               <div class="slider3__main_right">
                  <div class="slider3__main_block_logo"><img src="img/slider3/about3.jpg.jpg" alt=""></div>
               </div>
            </div>
            <div class="slider3__main_block blockn2">
               <div class="slider3__main_right">
                  <div class="slider3__main_block_logo"><img src="img/slider3/about4.jpg.jpg" alt=""></div>
               </div>
               <div class="slider3__block_left">
                  <div class="slider3__main_block_name block2">
                     <p>February 2014
                     </p>
                     <p>Our Triumph</p>
                  </div>
                  <div class="slider3__main_block_text block2">
                     <p>Proin iaculis purus consequat sem cure </p>
                     <p>digni ssim. Donec porttitora entum suscipit </p>
                     <p>aenean rhoncus posuere odio in tincidunt. Proin</p>
                     <p>iaculis purus consequat sem cure digni </p>
                     <p>ssim. Donec porttitora entum suscipit.</p>
                  </div>
               </div>

            </div>
         </div>
      </div>
      <div class="footer">
         <div class="footer__text">&copy; Copyright 2014 FreebiesXpress.com</div>
      </div>
   </div>
</body>

</html>